README.md

```diff
@@ -388,58 +392,55 @@ cd ~/eos/build/programs/eosd/
 <a name="walletimport"></a>
 ### Setting up a wallet and importing account key 
 
-Before running API commands you need to import the private key of an account you will be authorizing the transactions under into the EOS wallet.
-
 As you've previously added `plugin = eos::wallet_api_plugin` into `config.ini`, EOS wallet will be running as a part of `eosd` process.
 
-For testing purposes you can use a pre-created account `inita` from the `genesis.json` file.
-
-To login you need to run a command importing an active (not owner!) private key from `inita` account (you can find it in `~/eos/build/programs/eosd/data-dir/config.ini`) to the wallet.
+First we need to create a wallet, or load 
 
 ```bash
 cd ~/eos/build/programs/eosc/
 ./eosc wallet create # Outputs a password that you need to save to be able to lock/unlock the wallet
-./eosc wallet import 5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3
 ```
 
-Now you can issue API commands under `inita` authority.
+For the purpose of this walkthrough, we'll import the private key of the `inita` account so that you can issue API commands under authority of an existing account. This private key can be found within your `config.ini` and is provided for testing purposes. 
+
+```./eosc wallet import 5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3```
 
 <a name="createaccounts"></a>
-### Creating accounts for your smart contracts 
+### Creating accounts for sample "currency" contract
```

Comment:
it's one account I think?
also, should we here explain maybe that each contract should have an account associated with it?

Reply:
This is explained in General Use documentation in progress.